### ISSUE TITLE
Latest FutureMC Version Not Working With These Mods

### ISSUE THREAD

**Issue by DiamondPandaDP** [thedarkcolour/Future-MC#176]
![image](https://user-images.githubusercontent.com/60081863/87240210-18d4a800-c3dd-11ea-9e0f-f5e094d48c19.png)

**Comment by pinkpig3777** [comment]
To clarify are you saying it does not work with _any_ of those mods individually? Or that one of those mods is causing a problem?

Also you'll want to provide more detail in order to get support most likely. 
forge version:
mod version: (say the actual version not just 'latest')
latest.log: 
debug.log:
crash log (if applicable):

If it is one (or more) of those mods causing issues you can determine which is the problem mod with these steps:
1. Remove 1/2 of the mods.
2. See if it is fixed.
- If so, that means one of the removed mods is the problem and you should re-add 1/2 of the removed mods until you can narrow down the issue.
- If it is not fixed that means it is one of the still added mods and you should remove 1/2 of the remaining mods until you can narrow down the issue.

In other words, apply the Binary Search algorithm to quickly find the problem mod.

**Comment by juliantcf** [comment]
I can confirm FutureMC 1.12.2 works fine with Aether_Legacy, BetterFPS, JEI, Optifine, HarvestCraft, Quark, and TwilightForest. Latest versions (July 19th, 2020)

**Comment by thedarkcolour** [comment]
Which mods don't work?

**Comment by thedarkcolour** [comment]
Also pretty sure that orespawn doesn't have a 1.12 version

**Comment by HassaFelipe** [comment]
I can confirm that FutureMC doesn't really work well with Patchouli, it stops generating villages.

**Comment by HassaFelipe** [comment]
> I can confirm that FutureMC doesn't really work well with Patchouli, it stops generating villages.
Today I tested version 1.16-36-FABRIC of Patchouli and noticed that the generation of villages is working normally.

**Comment by thedarkcolour** [comment]
What... Future MC isn't for 1.16 or Fabric lmao

**Comment by TheMCSpacePro** [comment]
I think the 'Ore Spawn' mod was this: https://www.curseforge.com/minecraft/mc-mods/mmd-orespawn

**Comment by thedarkcolour** [comment]
I'm going to close this if no info can be given.
